iroha_python/src/lib.rs

```diff
@@ -0,0 +1,195 @@
+// Allow panic because of bad and unsafe pyo3
```

Comment:
I think you need to add some top level docs to this lib and `generator` lib and `iroha2_sys`. Right now it is a bit difficult to understand exact functions of each of them.